Multisig/UI/Settings/OwnerKeyManagement/AddKeyAsNewOwner/AddOwnerFlowFromSettings.swift

```diff
@@ -76,6 +77,7 @@ class AddOwnerFlowFromSettings: AddOwnerFlow {
 
     override func success() {
         assert(transaction != nil)
+        // TODO remove this here?
         AddressBookEntry.addOrUpdate(newOwner!.checksummed, chain: safe.chain!, name: newAddressName!)
```

Comment:
I think TODO is correct, if we add it earlier, then updating name here is not needed